evennia/objects/objects.py

```diff
@@ -1476,6 +1476,7 @@ def get_numbered_name(self, count, looker, **kwargs):
                 method is used.
             return_string (bool): If `True`, return only the singular form if count is 0,1 or
                 the plural form otherwise. If `False` (default), return both forms as a tuple.
+            no_article (bool): If `True`, do not return an article if `count` is 1.
```

Comment:
With this change, I think it would be suitable to also add an `Example:` block to this doc string to exemplify the use cases. the "The One Ring" example as well as "an Obj" vs just "Obj" are relevant.